File: sla/sla.rb
print "Digite o primeiro numero: "
num1 = gets.to_f

print "Digite o operador (+, -, *, /): "
op = gets.chomp

print "Digite o segundo numero: "
num2 = gets.to_f

if op == "+"
  resultado = num1 + num2
elsif op == "-"
  resultado = num1 - num2
elsif op == "*"
  resultado = num1 * num2
elsif op == "/"
  if num2 != 0
    resultado = num1 / num2
  else
    puts "Erro: divisao por zero!"
    exit
  end
else
  puts "Operador invalido!"
  exit
end

puts "Resultado: #{resultado}"

File: sla/sla.lua
io.write("Digite o primeiro numero: ")
local num1 = tonumber(io.read())

io.write("Digite o operador (+, -, *, /): ")
local op = io.read()

io.write("Digite o segundo numero: ")
local num2 = tonumber(io.read())

local resultado

if op == "+" then
    resultado = num1 + num2
elseif op == "-" then
    resultado = num1 - num2
elseif op == "*" then
    resultado = num1 * num2
elseif op == "/" then
    if num2 ~= 0 then
        resultado = num1 / num2
    else
        print("Erro: divisao por zero!")
        os.exit()
    end
else
    print("Operador invalido!")
    os.exit()
end

print("Resultado: " .. resultado)
